### ISSUE TITLE
No progress bar is displayed

### ISSUE THREAD

**Issue by q629988171** [CrypticSignal/better-ffmpeg-progress#20]
Ubuntu 20.04.4 LTS
PyCharm 2023.3.4 (Community Edition)
Python 3.8.10

The duration of input.mp4 has been detected as 144.033 seconds.

```python
from better_ffmpeg_progress import FfmpegProcess

# Pass a list of FFmpeg arguments, like you would if using subprocess.run()
process = FfmpegProcess(["ffmpeg", "-i", "input.mp4", "output.avi"])
# Use the run method to run the FFmpeg command.
process.run()
```

**Comment by stephanedebove** [comment]
Same problem here, on google colab. No progress bar is displayed, only the duration of my input video. @q629988171 you found a solution?

**Comment by CrypticSignal** [comment]
@q629988171 @stephanedebove Fixed in v2.1.3, you can upgrade Better FFmpeg Progress to the latest version with `pip3 install better-ffmpeg-progress` and you should no longer run into this issue. Let me know if this is not the case.

**Comment by stephanedebove** [comment]
Yes it works now thanks!
